File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/Java/A-Sheet #1-Data type/Z.java
import java.util.Scanner;

  public class Z {
  public static void main(String[] args){
  Scanner input = new Scanner(System.in);
  double num1,num2,num3,num4;
  num1 = input.nextDouble();
  num2 = input.nextDouble();
  num3 = input.nextDouble();
  num4 = input.nextDouble();

    if((num2*Math.log(num1)) > (num4*Math.log(num3))){
    System.out.println("YES");
  } else {
    System.out.println("NO");
  }
  input.close();
  }
}

File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/C/C-Sheet #2-Loop/Z.c
#include <stdio.h>

int main() {
  int num1,num2,counter = 0;
  scanf("%d %d",&num1,&num2);

  for (int i = 0;i <= num1;i++){
      for (int z = 0;z <= num1;z++){
          if (num2 - i - z >= 0 && num2 - i -z <= num1){
              counter++;
          }
      }
  }
  printf("%d\n",counter);

  return 0;
}
